src/snowcli/snow_connector.py

```diff
@@ -28,15 +26,10 @@ class SnowflakeConnector:
 
     def __init__(
         self,
-        snowsql_config: SnowsqlConfig,
-        connection_name: str,
+        connection_config: dict,
```

Comment:
Changed it but don't like it. I think it should be dataclass, because now we don't know what should be int this dict. WDYT?

Reply:
I agree with you but I don't think your changes are making it worse. Do you think so?

Reply:
Yes, but @sfc-gh-turbaszek PR will fix it :)